.github/workflows/ensure-jdk8.yml

```diff
@@ -0,0 +1,60 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Apache Netbeans
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  jdk11-jdk8:
+    name: "Compile with JDK-11 and test something on JDK-8"
+    runs-on: ubuntu-18.04
+    env:
+      OPTS: -Dcluster.config=basic -Dtest-unit-sys-prop.ignore.random.failures=true
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Caching dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/.hgexternalcache
+          key: ${{ runner.os }}-${{ hashFiles('**/external/binaries-list') }}
+          restore-keys: ${{ runner.os }}-
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+            distribution: 'zulu'
+            java-version: 8
+
+      - name: Record JDK8
+        run: echo "$JAVA_HOME" | tee `pwd`/jdk8
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+            distribution: 'zulu'
+            java-version: 11
+
+
+      - name: Build
+        run: ant $OPTS build
+
+      - name: Test java.source.base on JDK-8
```

Comment:
The future plan is: Once we discover another module that has this kind of compilation problem, we should add another test run here by duplicating line 59 and 60.

Reply:
@JaroslavTulach wouldn't a future plan of building on JDK 11, testing on JDK 8, for all JDK 8 tests be a better option?  At this point, that seems the more important combination to test.  If capacity an issue, could run a build on JDK 8 test on a regular basis, but not per PR?